kpi/context_processors.py

```diff
@@ -40,6 +41,32 @@ def get_mfa_enabled():
     }
 
 
+def custom_label_translations(
+    request, metadata_configs: dict, default_labels: dict
+):
+    """
+    Returns custom labels and translations for the signup form
+    from the USER_METADATA_FIELDS constance config.
+    """
+
+    # Get User Metadata Fields
+    loaded_metadata_fields = json.loads(metadata_configs)
```

Comment:
FYI, I resolved conflicts  with `beta` because I wanted it merged inside your branch to run it locally. 
Please ensure, I did not make any mistakes when I merged it. thank you. 🙏

Reply:
@JacquelineMorrissette Let's do a little refactor for that and move the logic inside the `I18nUtils` class.